bedrock/firefox/templates/firefox/developer/whatsnew-mdnplus.html

```diff
@@ -35,8 +35,8 @@ <h1 class="c-mdn-header-title">{{ ftl('developer-mdnplus-more-mdn-your-mdn') }}<
     </div>
 
     <div class="mandala-wrapper">
-      <div id="mandala" class="mandala-container animate-colors" aria-hidden="true">
-        <div class="mandala-translate mandala-rotate">
```

Comment:
@maureenlholland FYI I'm fiddling with a change that would only animate the mandala on body:hover, and stop after a minute and half: https://github.com/janbrasna/bedrock/commit/56cd3e1cbe0d8ad73ee7a1b3c5b8494198e132ed

(in case that's worth keeping, but cut rather short… cuz the rotation is 25-30% CPU, the color fade 30-35%; while the GPU for rotation has been already optimized, the energy impact of the color fade is still "orders of magnitude":/…)